jujugui/static/gui/src/app/components/revenue-statement/revenue-statement.js

```diff
@@ -0,0 +1,255 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+
+const Spinner = require('../spinner/spinner');
+const BasicTable = require('../basic-table/basic-table');
+
+/**
+  React component used to display a list of the users revenue-statements in their profile.
+*/
+class RevenueStatement extends React.Component {
+  constructor() {
+    super();
+    this.xhrs = [];
+    this.state = {
+      data: [
+        {
+          id: 1
+        },
+        {
+          id: 2
+        }
+      ],
+      loading: false
+    };
+  }
+
+  render() {
+    let content;
+    if (this.state.loading) {
+      content = <Spinner />;
```

Comment:
You no longer need to wrap elements when returning from react render so you can just return these content values